README.md

```diff
@@ -1,6 +1,7 @@
 # Quiz App
```

Comment:
I have adjusted this markdown file in accordance with these [Markdown Rules](https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md)